auth/server/src/main/java/com/seagates3/controller/PolicyController.java

```diff
@@ -160,8 +162,15 @@ public ServerResponse create() {
      */
     @Override public ServerResponse list() {
       List<Policy> policyList = null;
+      Map<String, Object> parameters = new HashMap<>();
+      if (requestBody.containsKey("OnlyAttached")) {
+        parameters.put("OnlyAttached", requestBody.get("OnlyAttached"));
```

Comment:
what parameters are these... for what this is being used. Name of variable does not clear it. Are they query parameters.. what is the purpose of it

Reply:
fixed